airgun/views/virtwho_configure.py

```diff
@@ -19,13 +21,52 @@
 )
 
 
+class StatusIcon(GenericLocatorWidget):
+    """The status for virtwho configure can be: green, gray.
+
+    green: The virt-who report has not arrived within the interval,
+         which indicates there was no change on hypervisor
+    gray: The configuration was not deployed yet or the virt-who was
+        unable to report the status
+    """
+
+    def __init__(self, parent, locator=None, logger=None):
```

Comment:
Do you need this `__init__` at all? Can't you just define your icon locator as class variable? Look at [`RadioGroup` in `airgun/widgets.py`](https://github.com/SatelliteQE/airgun/blob/master/airgun/widgets.py#L66) (or pretty much anything else inheriting from `GenericLocatorWidget` in that file).